packages/astro/package.json

```diff
@@ -153,14 +153,15 @@
     "mime": "^3.0.0",
     "network-information-types": "^0.1.1",
     "ora": "^6.3.1",
-    "sharp": "^0.32.1",
     "p-limit": "^4.0.0",
     "path-to-regexp": "^6.2.1",
     "preferred-pm": "^3.0.3",
     "prompts": "^2.4.2",
     "rehype": "^12.0.1",
     "semver": "^7.5.3",
+    "seroval": "^0.9.0",
```

Comment:
I am uncomfortable using a library that has not reached `1.0` yet.  

In the description, you say it's "battle-tested", but it's only used by solid. Could you please address my concerns?

Reply:
I am concerned by seroval's [use of `eval()`](https://github.com/lxsmnsyc/seroval/blob/v0.9.0/packages/seroval/src/index.ts#L100). Was [devalue](https://github.com/rich-harris/devalue) considered? It seems to be the SvelteKit equivalent.

Reply:
Do you think we could ask them to release a `1.0` version? Or at least ask them what's their plan with versioning.

Reply:
Looks like there are maybe some super-powerful streaming features planned prior to v1.0, but it's otherwise stable. https://twitter.com/lxsmnsyc/status/1689294233272737793

Reply:
Would that be non-breaking?

Reply:
Yes, I think there are separate APIs for any `async` or `streaming` operations